### PR TITLE
[17831] Release 2.8.2 EOL

### PR DESCRIPTION
**Pull request by jsantiago-eProsima**

### CODE REVIEW

File: docs/notes/notes.rst [hunk 1]
```diff
@@ -5,34 +5,46 @@
...
-    1. Include missing ReadCondition header.
-    2. Fix selection of output locators.
-    3. Fix null-dereference on ``parseXMLEnumDynamicType``.
-    4. Include ``2.8.x`` branch release support.
-    5. Send GAPs correctly when using separate sending.
-    6. Fixes for building in old compilers.
-    7. Fix |DataReader::read_next_instance-api| and |DataReader::take_next_instance-api| implementation.
+1. Fix bug in Topic creation with different Type Name
+2. Fix tests failing with subprocess aborted error
+3. Add ASAN CI tests for Fast DDS and Discovery Server
```
Comment (Mario-DL) [inline docs/notes/notes.rst:27]:
```suggestion
3. Fix communication with asymmetric ignoreParticipantFlags
```

**Review by Mario-DL — CHANGES_REQUESTED** [review]

**Comment by Mario-DL** [comment]
@richiprosima please test this

**Comment by EduPonz** [comment]
CI run:
[Build Status](http://jenkins.eprosima.com:8080/view/Fast%20DDS/job/fastdds_docs_manual/706/)

**Review by Mario-DL — APPROVED** [review]
All tests passed

**Review by EduPonz — APPROVED** [review]

**Pull request merged**